### PR TITLE
added pastas recipe

### PR DESCRIPTION
**Pull request by jentjr**

### CODE REVIEW

File: recipes/pastas/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 374f63736150cf214da2a7024923d65cd24189aa67275196d62663336b44e1f8
+
+build:
```
Comment (grst) [inline recipes/pastas/meta.yaml]: Can this be a `noarch:python` package? 
https://conda-forge.org/docs/meta.html#building-noarch-packages

File: recipes/pastas/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  number: 0
+  skip: True  # [py<35]
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+
+requirements:
+  host:
+    - matplotlib >=2.0
```
Comment (grst) [inline recipes/pastas/meta.yaml]: is matplotlib needed at build time? Also scipy/pandas?

File: recipes/pastas/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    - numpy >=1.10
```
Comment (grst) [inline recipes/pastas/meta.yaml]: please use `pin_compatible` for numpy

File: recipes/pastas/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  imports:
+    - pastas
+    - pastas.io
+    - pastas.project
+    - pastas.read
+    - pastas.stats
+
+about:
+  home: https://github.com/pastas/pastas
+  license: MIT
```
Comment (grst) [inline recipes/pastas/meta.yaml]: please add license file.

**Comment by jentjr** [comment]
@grst thanks for the review. I made all the changes.

**Comment by grst** [comment]
@conda-forge/staged-recipes, LGTM now

**Comment by jentjr** [comment]
Thanks!

**Pull request merged**